### ISSUE TITLE
Naming issue: vprm classes

### ISSUE THREAD

**Issue by User02468** [tglauch/pyVPRM#6]
Problem: Differences/purposes of classes vprm (VPRM.py) and vprm_base (vprm_base.py) are not obvious from names.
Proposal: Rename vprm to vprm_preprocessor (VPRM.py)
Additionally it might also be more understandable to rename vprm_base to vprm_model_base (and other vprm_model classes equivalently) to communicate that this is where the VPRM model equation is fitted, used etc.

**Comment by tglauch** [comment]
True! This has been resolved in the main branch now and will be pushed soon with the next release.
